tests/test_match.py

```diff
@@ -6,7 +6,9 @@
 import math
 import random
 from graspologic.match import GraphMatch as GMP
-from graspologic.simulations import er_np
```

Comment:
We should take a look at restructuring the tests folder -- this is in the root directory but would probably be better if our tests structure resembles the structure of the modules in graspologic. graspologic/match/qap -> tests/match/test_match vs tests/test_match. Shouldn't hold up this pr of course.